docs/source/Advanced-Usage.md

```diff
@@ -158,6 +160,11 @@ runtime_configuration = {
 pyintelowl_client.send_file_analysis_request(..., runtime_configuration=runtime_configuration)
 ```
 
+## Generic Analyzers
```

Comment:
adding some documentation about this is a good idea but I would change something because the list of generic analyzers was already inserted in `Usage.md` and people could wrongly understand that WiGLE is the only `generic analyzer`. I think you could just move this explanation above together with what you have already written in line 137. In this way a potential user would find all the required information in the same place